app/controllers/hostgroups_controller.rb

```diff
@@ -76,11 +75,10 @@ def destroy
   end
 
   def current_parameters
-    Taxonomy.as_taxonomy @organization, @location do
```

Comment:
Should this context be set still, since the taxonomy_scope filter is called and you're doing a Hostgroup lookup (authorisation)?